router/router.go

```diff
@@ -0,0 +1,51 @@
+package router
+
+import (
+	"Stock_broker_application/src/app/authentication/constants"
+	"Stock_broker_application/src/app/authentication/handlers"
+	"Stock_broker_application/src/app/authentication/repo"
+	"Stock_broker_application/src/app/authentication/service"
+
+	"github.com/gin-gonic/gin"
+	swaggerFiles "github.com/swaggo/files"
+	ginSwagger "github.com/swaggo/gin-swagger"
+)
+
+// SetUpRouter sets up the Gin router with API endpoints and Swagger documentation.
+// @title Stock Broker Application
+// @version 1.0
+// @description This is a Stock Broker Application API
+// @host localhost:8080
+// @BasePath /
+func SetUpRouter() *gin.Engine {
+	router := gin.Default()
+
+	// Define route for handling POST requests to create new users
+	// @Summary Create a new user
+	// @Description Handles HTTP POST request to create new users
+	// @Accept json
+	// @Produce json
+	// @Router /users [post]
+	router.POST(constants.CreateUserRoute, handlers.PostUsersData)
```

Comment:
keep all the router.POST together in code block like

router.POST(constants.CreateUserRoute, handlers.PostUsersData)
router.POST(constants.SignInRoute, handlers.SignInHandler(signInService))